pkg/webhook/mutation/pod/mutator/otlp/exporter/injectors.go

```diff
@@ -33,8 +33,9 @@ func (ti *traceInjector) Inject(c *corev1.Container, apiURL string, override boo
 		return false
 	}
 
-	addEnvVarLiteralValue(c, OTLPTraceEndpointEnv, fmt.Sprintf("%s/%s", apiURL, "traces"))
+	addEnvVarLiteralValue(c, OTLPTraceEndpointEnv, fmt.Sprintf("%s/%s", apiURL, "v1/traces"))
```

Comment:
only one of the `%s` is actually variable. the other can be part of the format string